R/initialise.R

```diff
@@ -11,9 +11,9 @@
 #'   system.file("examples", "dettl_config.yml", package = "dettl"),
 #'   add_log_table = FALSE
 #' )
-#' dettl_initialise(file.path(path, "person_information"), "test")
+#' dettl_db_create_log_table(file.path(path, "person_information"), "test")
```

Comment:
what do you think about `dettl::` prefixes to all examples?  They run with the package attached so it's not strictly necessary but it seems that's how we'd tend to use the package.  Should be followed through the vignettes (which currently do use `dettl::`) and (eventually) in any examples that end up in the README